From 4fd1ea5f87ca629d43d01eb275cbeda9539f22c8 Mon Sep 17 00:00:00 2001
From: Shuwei Wu <shuwei.wu@mailbox.org>
Date: Fri, 10 Apr 2026 15:58:22 +0800
Subject: [PATCH 141/281] FROMLIST: cpufreq: dt-platdev: Add SpacemiT K1 SoC to
 the allowlist

The SpacemiT K1 SoC uses standard device tree based CPU frequency
scaling. Add it to the allowlist to instantiate the cpufreq-dt driver.

Signed-off-by: Shuwei Wu <shuwei.wu@mailbox.org>
Link: https://lore.kernel.org/r/20260410-shadow-deps-v2-1-4e16b8c0f60e@mailbox.org
Signed-off-by: Han Gao <gaohan@iscas.ac.cn>
---
 drivers/cpufreq/cpufreq-dt-platdev.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/cpufreq/cpufreq-dt-platdev.c b/drivers/cpufreq/cpufreq-dt-platdev.c
index 25fd3b191b7e..31a64739df25 100644
--- a/drivers/cpufreq/cpufreq-dt-platdev.c
+++ b/drivers/cpufreq/cpufreq-dt-platdev.c
@@ -81,6 +81,7 @@ static const struct of_device_id allowlist[] __initconst = {
 		{ .have_governor_per_policy = true, },
 	},
 
+	{ .compatible = "spacemit,k1", },
 	{ .compatible = "st-ericsson,u8500", },
 	{ .compatible = "st-ericsson,u8540", },
 	{ .compatible = "st-ericsson,u9500", },
-- 
2.47.3

